tests/integration/_experimental/test_quality_standard.py

```diff
@@ -0,0 +1,100 @@
+from tests.integration._experimental.helper import create_quality_standard
+from tests.integration.helper import fake_locator
+from tests.integration.helper import with_test_prefix
+
+N_DATAPOINTS = 20
+ID_FIELDS = ["locator"]
+
+
+@pytest.fixture
+def datapoint_df() -> pd.DataFrame:
+    return pd.DataFrame(
+        [
+            dict(
+                locator=fake_locator(i, "datapoints"),
+                city=["new york", "waterloo"][i % 2],
+            )
+            for i in range(N_DATAPOINTS)
+        ],
+    )
+
+
+@pytest.fixture
+def result_df() -> pd.DataFrame:
+    return pd.DataFrame([dict(locator=fake_locator(i, "datapoints"), score=i * 0.1) for i in range(N_DATAPOINTS)])
+
+
+def test__download_quality_standard_result(datapoint_df: pd.DataFrame, result_df: pd.DataFrame) -> None:
+    dataset_name = with_test_prefix("test__download_quality_standard_result__dataset")
+    model_name = with_test_prefix("test__download_quality_standard_result__model")
+    upload_dataset(dataset_name, datapoint_df, id_fields=ID_FIELDS)
+    _upload_results(
+        dataset_name,
+        model_name,
+        result_df,
+    )
+
+    metric_group_name = "test group"
+    metric_name = "Min Score"
+    test_case_name = "city"
+    quality_standard = dict(
+        name=with_test_prefix("test__download_quality_standard_result__quality_standard"),
+        stratifications=[
+            dict(
+                name=test_case_name,
+                stratify_fields=[dict(source="datapoint", field="city", values=["new york", "waterloo"])],
+                test_cases=[
+                    dict(name="new york", stratification=[dict(value="new york")]),
+                    dict(name="waterloo", stratification=[dict(value="waterloo")]),
+                ],
+            ),
+        ],
+        metric_groups=[
+            dict(
+                name=metric_group_name,
+                metrics=[dict(label=metric_name, source="result", aggregator="min", params=dict(key="score"))],
+            ),
+        ],
+        version="1.0",
+    )
+    create_quality_standard(dataset_name, quality_standard)
+
+    quality_standard_df = download_quality_standard_result(dataset_name, [model_name])
+    assert_frame_equal(
+        quality_standard_df,
+        download_quality_standard_result(dataset_name, [model_name], [metric_group_name]),
+    )
+
+    df_columns: pd.MultiIndex = quality_standard_df.columns
+    assert df_columns.names == ["model", "eval_config", "metric_group", "metric"]
+    assert df_columns.levels == [[model_name], ["null"], [metric_group_name], [metric_name]]
+
+    df_index: pd.MultiIndex = quality_standard_df.index
+    assert df_index.names == ["stratification", "test_case"]
+    assert all(df_index.levels[0] == ["Dataset", test_case_name])
+    assert all(df_index.levels[1] == ["new york", "waterloo"])
+
+    assert quality_standard_df.loc[("Dataset", np.nan), (model_name, "null", metric_group_name, metric_name)] == 0.0
```

Comment:
string "null" is expected? how's this look like for a non-null eval config?

Reply:
A json blob. I'll update the test to add a second model with an eval config

Reply:
Pandas is stringifying the null here, but yes it is expected to be an empty cell if there is no eval config